goodDeeds/app/src/main/java/com/goodpeople/gooddeeds/model/GoodDeeds.java

```diff
@@ -222,8 +222,7 @@ public void editOffer(String subject, String description) {
      * @return all deeds as IDeed
```

Comment:
If we change this we need to change javadoc.

Reply:
dont understand :O